src/components/News.js

```diff
@@ -4,75 +4,68 @@ import PropTypes from "prop-types";
 import InfiniteScroll from "react-infinite-scroll-component";
 import Loader from "./Loader";
 import { API_KEY, BASE_API_URL } from "../constants";
+import { capitalize } from "../utils";
 
-const News = (props) => {
+const News = React.memo((props) => {
```

Comment:
I was not knowing about React.memo before, Thank you for this commit i realized it can really help our application overall performance